### PR TITLE
Add go-reportcard and godoc badges

### PR DESCRIPTION
**Pull request by charlesakalugwu**
Added the go report card and godoc badges to the project readme. 

I'll also try to get the codecov badge + integration in this week.

/cc @openshift/sig-azure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## openshift-azure
 
+[![Go Report Card](https://goreportcard.com/badge/github.com/openshift/openshift-azure)](https://goreportcard.com/report/github.com/openshift/openshift-azure)
```
Comment (kargakis) [inline README.md]: Returns a 404, is this expected?

Comment (kargakis) [reply README.md]: Sorry, I opened a bad link

/lgtm

**Comment by mjudeikis** [comment]
Awesome :)

**Pull request merged**